ddtrace/internal/telemetry/writer.py

```diff
@@ -54,11 +55,14 @@ def __init__(self, agent_url=None):
         self._integrations_queue = []  # type: List[Dict]
         self._lock = forksafe.Lock()  # type: forksafe.ResetObject
         self._forked = False  # type: bool
+        # Debug flag that enables payload debug mode.
+        self._debug = asbool(os.environ.get("DD_TELEMETRY_DEBUG", "false"))
         forksafe.register(self._fork_writer)
 
         self._headers = {
             "Content-type": "application/json",
             "DD-Telemetry-API-Version": "v1",
+            "DD-Telemetry-Debug-Enabled": str(self._debug).lower(),
```

Comment:
Can we make `self._debug` a string? Do we plan to use it as a boolean?

Reply:
@mabdinur I'm curious to understand why it's not acceptable for `self._debug` to be a boolean.

Reply:
It can be a boolean but right now it's only used as a string

Reply:
We need a bool value type here -> https://github.com/DataDog/dd-trace-py/pull/5020/files#diff-0cb9315daf5ed5208de657467cc10a6b7dda616a161c9b5b1f8cff83c95894deR261